src/WebJobs.Extensions.DocumentDB/Trigger/CosmosDBTriggerAttribute.cs

```diff
@@ -77,5 +77,19 @@ public CosmosDBTriggerAttribute(string databaseName, string collectionName)
         /// Gets or sets the lease options for this particular DocumentDB Trigger. Overrides any value defined in <see cref="DocumentDBConfiguration" /> 
```

Comment:
This is unrelated to this review -- but could you update the summary for `LeaseCollectionName` to note that the default value is "leases"?